src/test/java/pages/SmartphonePage.java

```diff
@@ -0,0 +1,63 @@
+package pages;
+
+import elements.Text;
+import indices.Pages;
+import org.openqa.selenium.By;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.testng.Assert;
+import properties.Property;
+import util.ScrollUtils;
+import util.WaitUtils;
+
+
+public class SmartphonePage extends BasePage {
+
+    public static final String MOBILE_ENDPOINT = "mobile";
+
+    @FindBy(xpath = "//h1[@class='schema-header__title']")
+    protected Text phonesTitle;
+
+    protected static final String manufacturer = "//div[6]//li//input[@value= '%s']/../span";
```

Comment:
does it need to be protected? and the element above?

Reply:
changed